lib/internal/process/esm_loader.js

```diff
@@ -58,15 +59,41 @@ async function initializeLoader() {
   const { getOptionValue } = require('internal/options');
   const customLoaders = getOptionValue('--experimental-loader');
   const preloadModules = getOptionValue('--import');
-  const loaders = await loadModulesInIsolation(customLoaders);
+
+  let cwd;
+  try {
+    cwd = process.cwd() + '/';
+  } catch {
+    cwd = 'file:///';
```

Comment:
```suggestion
    cwd = '/';
```

Reply:
I reused the code from a bit below: https://github.com/nodejs/node/blob/98f477f53740f53a00bf75c9c5f7241baee92af8/lib/internal/process/esm_loader.js#L105-L110

Reply:
There may be a bug in the below code too, because `pathToFileURL('file:///').href` returns `file:///home/dir/file:/`